examples/40_paper/2018_kdd_rijn_example.py

```diff
@@ -42,8 +44,8 @@
 # Note that we simplify the example in several ways
 # 1) We only consider numerical hyperparameters
 # 2) We consider all hyperparameters that are numerical (in reality, some
-#    hyperparameters might be inactive (e.g., `degree`) or irrelevant (e.g.,
-#    `random_state`)
+#    hyperparameters might be inactive (e.g., ``degree``) or irrelevant
```

Comment:
There's a formatting issue here. Could you please add a newline before `1)` to make the ordered list start?

Reply:
how can I see the examples in this branch btw? I have no clue how to manually inspect the examples before merging into master, which might take a while

Reply:
Examples in this branch are not deployed online, you need to build them locally using `make html` in the `doc` directory.